src/Sulu/Bundle/AdminBundle/Resources/js/components/AutoComplete/Suggestion.js

```diff
@@ -0,0 +1,80 @@
+// @flow
+import React from 'react';
+import type {Node} from 'react';
+import Icon from '../Icon';
+import suggestionStyles from './suggestion.scss';
+
+type Props = {
+    value: string,
+    query: string,
+    icon?: string,
+    children: Node | (highlight: (text: string) => Node) => Node,
```

Comment:
Shouldn't it be `string` instead of `Node`? `Node` would mean that you can also pass `div`s, `span`s and so on, that's what we wanted to avoid, isn't it?